src/lib/network/network_autoyast.rb

```diff
@@ -120,6 +117,26 @@ def set_network_service
       NetworkService.EnableDisableNow
     end
 
+    # Checks if given profile contains device names in old style format
+    #
+    # OLd style device name format is eth-id-<mac> or eth-bus-<busid>.
+    # These names has to be converted into new-style (eth0) and name persistency
+    # is enabled using udev rule which is generated later.
+    #
+    # @param [Hash] configuration map
+    #
+    # @return true if configuration contains old style name
+    def oldStyle(ay)
```

Comment:
copy - pasted function. I've kept that until I grep some other modules if it is used.
Taking note to change it once done.